specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_Get.json

```diff
@@ -0,0 +1,73 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "amlFilesystemName": "fs1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "tags": {
+          "Dept": "ContosoAds"
+        },
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.StorageCache/amlFilesystems/fs1",
+        "location": "westus",
+        "name": "fs1",
+        "type": "Microsoft.StorageCache/amlFilesystem",
+        "properties": {
+          "storageCapacityTiB": 3072,
+          "zones": ["1", "2"],
+          "health": {
+            "state": "Healthy",
+            "statusDescription": "amlFilesystem is ok."
+          },
+          "provisioningState": "Succeeded",
+          "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/amlvnet/subnets/sub1",
+          "encryptionSettings": {
+            "keyEncryptionKey": {
+              "keyUrl": "https://examplekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+              "sourceVault": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/examplekv"
+            }
+          },
+          "maintenanceWindow" : {
+            "dayOfWeek": "Friday",
+            "timeOfDay" : "22:00"
+          },
+          "maintenanceStatus": {
+            "currentFirmwareVersion": "5.3.23",
+            "softwareUpdateStatus": "available"
+          },
+          "hsmSettings": {
+            "container": "https://somecontainer.blob.core.windows.net/containername",
+            "containerKeyVaultKey": {
+              "keyUrl": "https://somekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+              "sourceVault": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/somekv"
+            },
+            "importPrefix": "aml",
+            "exportPrefix": "aml"
+          },
+          "namespaceHydrationStatus": {
```

Comment:
Should we also return a detailed health status in the response?  If cluster is in state "Degraded" (for example) then it might be useful to know the Ops view of why the cluster is in degraded state.

Reply:
"health": {
            "state": "Healthy",
            "statusDescription": "amlFilesystem is ok."
          },

the health information is a few lines earlier.
It has the definition 
"amlFilesystemHealth": {
      "description": "An indication of amlFilesystem health. Gives more information about health than just that related to provisioning.",
      "properties": {
        "state": {
          "description": "List of amlFilesystem health states.",
          "enum": [
            "Unknown",
            "Healthy",
            "Down",
            "Transitioning",
            "Maintenance"
          ],
          "x-ms-enum": {
            "name": "amlFilesystemHealthStateType",
            "modelAsString": true
          },
          "type": "string"
        },
        "statusDescription": {
          "description": "Describes of state.",
          "type": "string"
        }
      }